src/js/settings.js

```diff
@@ -117,5 +117,7 @@ function(L) {
     'fillColor': '#fcd96c'
   };
 
+  settings.cors = ('withCredentials' in new XMLHttpRequest());
```

Comment:
This syntax is a little funky. Can we not use hasOwnProperty? Also very possible I'm misreading the intent (see if the new XMLHttpRequest() has withCredentials?)